client/my-sites/post-type-list/post-actions-ellipsis-menu/copy-link.jsx

```diff
@@ -0,0 +1,59 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PopoverMenuItemClipboard from 'components/popover/menu-item-clipboard';
+import { getPost } from 'state/posts/selectors';
+import { bumpStat, recordTracksEvent } from 'state/analytics/actions';
+import { bumpStatGenerator } from './utils';
+
+function PostActionsEllipsisMenuCopyLink( { onCopyLinkClick, copyLink, translate } ) {
+	return (
+		<PopoverMenuItemClipboard text={ copyLink } onCopy={ onCopyLinkClick }>
+			{ translate( 'Copy Link', { context: 'verb' } ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **87** times:
`translate( 'Copy Link' )` *ES Score: 12*


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.